src/main/java/be/syntra/devshop/DevshopBack/services/DTOMapper.java

```diff
@@ -0,0 +1,16 @@
+package be.syntra.devshop.DevshopBack.services;
+
+import be.syntra.devshop.DevshopBack.entities.Product;
+import be.syntra.devshop.DevshopBack.models.ProductDTO;
+import org.springframework.stereotype.Component;
+
+
+@Component
+public class DTOMapper implements DTOMapperService {
+    public Product convertToProduct(ProductDTO productDTO) {
+        return Product.builder()
+                .name(productDTO.getName())
+                .price(productDTO.getPrice())
+                .build();
+    }
+}
```

Comment:
should be given a better name and its own package
this can be a service but then it should reside in a subPackage of services
or mappers should have it's own separate package